evo1/observer.py

```diff
@@ -2,23 +2,33 @@
 import logging
 
 from engine.mathlib import Vec2, dist
+from evo1.atb import SeqATBCombatManual, SeqATBmove2D
 from evo1.memory import get_zelda_memory
-from evo1.move2d import SeqSection2D
 from term.window import WindowLayout
 
 logger = logging.getLogger(__name__)
 
 
-class SeqObserver2D(SeqSection2D):
+class SeqObserver2D(SeqATBmove2D):
```

Comment:
This is a debugging class used for testing purposes. It's not used anywhere in the actual run.